yarn.lock

```diff
@@ -2832,6 +2832,14 @@
   resolved "https://registry.yarnpkg.com/@open-draft/until/-/until-1.0.3.tgz#db9cc719191a62e7d9200f6e7bab21c5b848adca"
   integrity sha512-Aq58f5HiWdyDlFffbbSjAlv596h/cOnt2DO1w3DOC7OJ5EHs0hd/nycJfiu9RJbT6Yk6F1knnRRXNSpxoIVZ9Q==
 
+"@opentripplanner/icons@1.2.4":
```

Comment:
You can remove the changes to this file from the PR. The underlying issue is handled by #444.

Reply:
Sounds good, I will revert it